doc/release/upcoming_changes/15685.new_feature.rst

```diff
@@ -0,0 +1,7 @@
+``subok`` option for `numpy.copy`
+---------------------------------
+A new kwarg, ``subok``, was added to `numpy.copy` to allow users to toggle the 
+behavior of `numpy.copy` with respect to array subclasses. The default value
+is ``False`` which is consistent with the behavior of `numpy.copy` for 
+previous numpy versions. To create a copy that preserves an array subclass with
+`numpy.copy`, call ``np.copy(arr, subok=True)``.
```

Comment:
Maybe add "which can be preferably spelled `arr.copy()` ?

Reply:
I added a sentence to the end of the release note to highlight the difference in the default behavior between `np.copy` and the `np.ndarray.copy` method. Let me know if that change was an improvement.